docs/dev/guidelines/code-review.rst

```diff
@@ -0,0 +1,62 @@
+******
+Code Review
+******
+
+STEPS
+======
+
+0. Precondition -> only Developer
+==================================
+
+* Limit yourself to one functionality per pull request.
+* Split up your task in multiple branches & pull requests if necessary.
+* `Commit Early, Commit Often, Perfect Later, Publish Once <https://speakerdeck.com/lemiorhan/10-git-anti-patterns-you-should-be-aware-of>`_.
+
+1. Start Implementation -> only Developer
+========================================
+
+* `Open draft pull request. <https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request>`_ Allows for code related questions and discussions.
+
+2. Implementation is "done" -> only Developer
+=============================================
+
+* Make sure all steps in the `Checklist <https://github.com/ls1intum/ArTEMiS/blob/develop/.github/PULL_REQUEST_TEMPLATE.md>`_ are completed.
+* Add or update the "Steps for Testing" in the description of your pull request.
+* Mark the pull request as `ready for review <https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request>`_.
+* Add Stephan Krusche (@krusche) and Jan Philip Bernius (@jpbernius) and your review partner as reviewers to the pull request.
+* Make sure that the changes in the pull request are only the ones necessary
+
+3. Review -> only Reviewer
+==========================
+
+* Perform the "Steps for Testing" and verify that the new functionality is working as expected.
+* Verify that related functionality is still working as expected.
+* Check the Changes to
+* Conform with the code style
+* Make sure you can easily understand the code
+* Make sure that (extensive) comments are present where deemed necessary
+* Performance is reasonable (e.g. Number of database queries or HTTP calls)
+* Submit your comments and status (Approve or Request Changes) using GitHub.
+
+4. Respond to review- > Developer & Reviewer
+=====================
```

Comment:
You missed a codacy issue here. I think you need to underline the title completely.